samcli/commands/local/cli_common/options.py

```diff
@@ -153,13 +153,25 @@ def warm_containers_common_options(f):
     warm_containers_options = [
         click.option(
             "--warm-containers",
-            help="Optional. Specifies how AWS SAM CLI manages containers for each function.",
+            help="""
+            \b
+            Optional. Specifies how AWS SAM CLI manages 
+            containers for each function.
+            Two modes are available:
+            EAGER: Containers for all functions are 
```

Comment:
I did not find our existing multi-line `help=`. What does the stdout look like here?

Reply:
it looks like this:
```
--warm-containers [EAGER|LAZY]    Optional. Specifies how AWS SAM CLI manages 
                                  containers for each function.
                                  Two modes are available:
                                  EAGER: Containers for all functions are 
                                  loaded at startup and persist between 
                                  invocations.
                                  LAZY:  Containers are only loaded when each 
                                  function is first invoked. Those containers 
                                  persist for additional invocations.
```

Reply:
this is pretty cool. Good to know, thx